trufflesuite/truffle
# Refactor @truffle/preserve-to-ipfs

- Update to the latest IPFS libraries
- Update other dependencies
- Extract IPFS interfaces from updated TS IPFS library where possible (not possible everywhere unfortunately)
- Update code to use refactored @truffle/preserves
- Separate fixture data from tests
- Test that the expected events are emitted
